changelog/2648.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added a new keyword ``particlewise`` to the function `~plasmapy.particles.particle_collections.ParticleList.is_category`,
```

Comment:
```suggestion
Added a new keyword ``particlewise`` to the method `~plasmapy.particles.particle_collections.ParticleList.is_category` of |ParticleList|,
```
🔧📚 Slight terminology change to reflect that this is a method attached to a class rather than a standalone function.

It'll also be helpful to explicitly state `ParticleList` here for context, since the `~` causes the namespace to be hidden in the rendered documentation so `ParticleList` itself won't appear in the changelog otherwise.